api/controllers/comment.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const winston = require('winston');
+const request = require('request');
+const axios = require('axios');
+
+const config = require('../config');
+const logger = winston.createLogger(config.logger.winston);
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/', common.jwt({credentialsRequired: true}), (req, res, next) =>{
+    if(!req.query.project) return next("project not defined");
+    let project = req.query.project;
+    if(project) {
+        const find = {"project": project, "removed" : false};
+        db.Comments.find(find).exec((err, recs)=>{
+            // console.log(recs);
+            if(err) return next(err);
+            // console.log(recs);
+            db.Comments.countDocuments(find).exec((err, count)=>{
```

Comment:
I don't think we need to return the total counts. You can just return the comments `res.json(recs)`. Can you limit the result to the *last* 100 comments?

Reply:
This line should be removed